<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="Ideas2IT" failures="3" tests="5" name="OneMG" time="982.618" errors="0" timestamp="04 Oct 2023 15:10:13 GMT">
  <testcase classname="com.leaftaps.testcases.Netmeds_CommonTests" name="runTests" time="51.047">
    <failure type="java.lang.IndexOutOfBoundsException" message="Index -1 out of bounds for length 0">
      <![CDATA[java.lang.IndexOutOfBoundsException: Index -1 out of bounds for length 0
	at java.base/jdk.internal.util.Preconditions.outOfBounds(Preconditions.java:64)
	at java.base/jdk.internal.util.Preconditions.outOfBoundsCheckIndex(Preconditions.java:70)
	at java.base/jdk.internal.util.Preconditions.checkIndex(Preconditions.java:266)
	at java.base/java.util.Objects.checkIndex(Objects.java:361)
	at java.base/java.util.ArrayList.get(ArrayList.java:427)
	at com.framework.selenium.api.base.SeleniumBase.periodicallyCheck_For_NewCardLoads(SeleniumBase.java:1206)
	at com.leaftaps.pages.MainPage_Netmeds.scrollTillLastMoleculeResults(MainPage_Netmeds.java:47)
	at com.leaftaps.testcases.Netmeds_CommonTests.runTests(Netmeds_CommonTests.java:26)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:539)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
	at java.base/java.lang.Thread.run(Thread.java:833)
... Removed 9 stack frames]]>
    </failure>
  </testcase> <!-- runTests -->
  <testcase classname="com.leaftaps.testcases.Netmeds_CommonTests" name="runTests" time="60.055"/>
  <testcase classname="com.leaftaps.testcases.Netmeds_CommonTests" name="runTests" time="918.707">
    <failure type="java.lang.RuntimeException" message="Test aborted">
      <![CDATA[java.lang.RuntimeException: Test aborted
	at com.leaftaps.pages.MainPage_Netmeds.scrollTillLastMoleculeResults(MainPage_Netmeds.java:54)
	at com.leaftaps.testcases.Netmeds_CommonTests.runTests(Netmeds_CommonTests.java:26)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:539)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
	at java.base/java.lang.Thread.run(Thread.java:833)
... Removed 9 stack frames]]>
    </failure>
  </testcase> <!-- runTests -->
  <testcase classname="com.leaftaps.testcases.Netmeds_CommonTests" name="runTests" time="921.915">
    <failure type="java.lang.RuntimeException" message="Test aborted">
      <![CDATA[java.lang.RuntimeException: Test aborted
	at com.leaftaps.pages.MainPage_Netmeds.scrollTillLastMoleculeResults(MainPage_Netmeds.java:54)
	at com.leaftaps.testcases.Netmeds_CommonTests.runTests(Netmeds_CommonTests.java:26)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:539)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
	at java.base/java.lang.Thread.run(Thread.java:833)
... Removed 9 stack frames]]>
    </failure>
  </testcase> <!-- runTests -->
  <testcase classname="com.leaftaps.testcases.Netmeds_CommonTests" name="runTests" time="939.606"/>
</testsuite> <!-- OneMG -->
